.github/workflows/publish-autoinstrumentation-nodejs.yaml

```diff
@@ -71,7 +71,7 @@ jobs:
         uses: docker/build-push-action@v5
         with:
           context: autoinstrumentation/nodejs
-          platforms: linux/amd64,linux/arm64
+          platforms: linux/amd64,linux/arm64,linux/s390x,linux/ppc64le
```

Comment:
I think we should do the nodejs bump and the new architecture in separate PRs.  That way if something goes weird with the new architectures again we don't have to roll back the nodejs bump, which is independent.

Reply:
Actually, the problem was the bump in the version of NodeJS. The platforms were not related.

Reply:
Then we should separate them to ensure the node bump works as expected this time before introducing new architectures.

Reply:
Since we've had 1 revert with this already, I'd prefer keeping the git history as clean as possible so any other reverts change only the necessary pieces.

Reply:
+1 on @TylerHelmuth